migration-guide/resources/app.md

```diff
@@ -2,7 +2,7 @@
 
 Provides a Cloud Foundry resource to manage applications.
 
-|  SAP Cloud Foundry Provider |Community Cloud Foundry Provider |
+| cloudfoundry/cloudfoundry |Community Cloud Foundry Provider |
```

Comment:
Shouldn't we follow the same convention of replacing the Community Cloud Foundry Provider header with cloudfoundry-community/cloudfoundry as provided in the migration guide of data sources?

Reply:
Good catch. Updated :)